src/SamplesApp/SamplesApp.Droid/SamplesApp.Droid.csproj

```diff
@@ -16,8 +16,7 @@
     <AndroidResgenFile>Resources\Resource.Designer.cs</AndroidResgenFile>
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <TargetFrameworkVersion>v12.0</TargetFrameworkVersion>
-		<AndroidUseLatestPlatformSdk>true</AndroidUseLatestPlatformSdk>
-		<AndroidManifest>Properties\AndroidManifest.xml</AndroidManifest>
```

Comment:
The IDE removes the property incorrectly, which makes Android builds on macOS fail. We'll make a separate change to prevent this from happening. For now, please keep this file as-is.